services/database/mail.ts

```diff
@@ -0,0 +1,37 @@
+import prisma from "@database";
+
+export default async function findEmails(hoursWithin: number) {
```

Comment:
Please add comments to describe what this function is doing, what it takes in, and what it returns.